src/views/results/Results.vue

```diff
@@ -227,6 +227,15 @@ export default {
             yAxes: input.yAxes,
             xAxes: [
               {
+                ticks: {
+                  autoSkip: false,
+                  maxTicksLimit: 20,
```

Comment:
Since we have more space between the ticks, you could make the text horizontal. Just add the following params.

```
maxRotation: 0,
minRotation: 0,
```

Reply:
You're right! it'll be better